File: DatingApp.API/Models/Message.cs
using System;

namespace DatingApp.API.Models
{
    public class Message
    {
        public int Id { get; set; } 
        public int SenderId { get; set; }   
        public User Sender { get; set; } 
        public User Recipient { get; set; } 
        public int RecipientId { get; set; }  
        public string Content { get; set; }
        public bool IsRead { get; set; }
        public DateTime? MessgaeReadTime { get; set; }
        public DateTime? MessageSentTime { get; set; }
        public bool RecipientDeleted { get; set; }
        public bool SenderDeleted { get; set; }
    }
}

File: DatingApp-SPA/src/app/_models/message.ts
export interface Message {
    id: number;
    senderId: number;
    senderName: string;
    senderPhotoUrl: string;
    recipientId: number;
    recipientName: string;
    recipientPhotoUrl: string;
    content: string;
    isRead: string;
    messgaeReadTime: Date;
    messageSentTime: Date;

}
